EasyPost.Tests/EndShipperTest.cs

```diff
@@ -1,93 +1,87 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
-using EasyPost.Beta;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using EasyPost.Models.API;
+using EasyPost.Utilities.Annotations;
+using Xunit;
 
 namespace EasyPost.Tests
 {
-    [TestClass]
-    public class EndShipperTest
+    public class EndShipperTest : UnitTest
     {
-        private TestUtils.VCR _vcr;
+        public EndShipperTest() : base("end_shipper", TestUtils.ApiKey.Production)
+        {
+        }
 
-        // This (and all Beta features) cannot use VCR because of a known conflict between using a VCR/custom HttpClient and re-constructing RestSharp.
```

Comment:
Question: Can you explain what this was? It's been awhile and I've forgotten. What allows us to be able to do this now?

Reply:
Because of the singleton client manager, we could not re-initialize the underlying RestSharp client, a step that was necessary to inject the EasyVCR HTTP client middleware. Because of the re-architecture, this is no longer the case.

Reply:
Ah yes. Nice!